File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
import java.util.*;

class Solution {
    public String solution(String s) {
        String[] str = s.split(" ");
        int[] intArray = new int[str.length];
        String answer =" ";
        
        for(int i=0; i<str.length;i++){
            intArray[i]=Integer.parseInt(str[i]);
        }        
        
        int max = intArray[0];
        int min = intArray[0];
        for(int i=0;i<intArray.length;i++){
           	max = Math.max(max, intArray[i]);
			min = Math.min(min, intArray[i]);
        }
        
        return min+answer+max;
    }
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    
    answer = s.split(' ')
    
    num_list=[]
    
    length = len(answer)
    
    for i in range(length):
        num_list.append(int(answer[i]))
    
    num_list.sort()
    
    test1 = list(map(int, s.split(' ')))
    print(test1)
    
    test2 = map(int, s.split(' ')) # map -> 원하는 타입으로 쉽게 변경가능
    print(test2)
    
    return str(num_list[0]) + " " + str(num_list[length-1])